src/app/shell/application/cookies-banner/cookies-banner.component.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { PLATFORM_ID } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { BrowserTransferStateModule } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { RouterTestingModule } from '@angular/router/testing';
+import { TranslateModule } from '@ngx-translate/core';
+import { MockDirective } from 'ng-mocks';
+import { instance, mock } from 'ts-mockito';
+
+import { ServerHtmlDirective } from 'ish-core/directives/server-html.directive';
+import { CookiesService } from 'ish-core/services/cookies/cookies.service';
+
+import { CookiesBannerComponent } from './cookies-banner.component';
+
+// tslint:disable:no-intelligence-in-artifacts
```

Comment:
I wasn't aware this rule is checking tests. Is it really?!

Reply:
![image](https://user-images.githubusercontent.com/50741106/96468805-cea96100-122c-11eb-8655-e2294fac8e11.png)
At least VS Code is applying the tslint rule to the `*spec.ts` as well.

Reply:
![image](https://user-images.githubusercontent.com/50741106/96470082-36ac7700-122e-11eb-9489-d3dfbf72f721.png)
`npm run lint` is not happy too.

Reply:
I double checked in tslint configuration: tests are included for this rule, so you'll need the ingore. 👍